html_css/intermediate_html/html-tables.md

```diff
@@ -25,17 +25,20 @@ In this codepen example we created a table element, containing two table row ele
 
 ### Assignment 
 <div class="lesson-content__panel" markdown="1">
-1. Read <a href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics">this basic</a> and <a href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced">this advanced</a> tutorial from MDN. It should teach you all the syntax behind it. It's pretty straightforward. Make sure to code along!
-2. Do <a href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Structuring_planet_data">this</a> assessment from MDN. It's good to put your newly learned skills to practice!
+1. Read [this basic](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics) and [this advanced](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced) tutorial from MDN. It should teach you all the syntax behind it. It's pretty straightforward. Make sure to code along!
+2. Do [this](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Structuring_planet_data) assessment from MDN. It's good to put your newly learned skills to practice!
 </div>
 
 ### Additional Resources
+This section contains helpful links to other content. It isn’t required, so consider it supplemental for if you need to dive deeper into something.
+
+- Pencil & Paper published [a great article about things you can achieve with tables](https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables) if you dare to spend some time giving them some love. They suggest some good habits on how you should format your data which is pretty simple and makes a huge difference. These are not a list of must does to a table, but ideas worth keeping in mind for the next time a table is bothering you with how it looks. 
```

Comment:
Suggestion on making the link text more accessible.

```suggestion
- Pencil & Paper published a great [article about things you can achieve with tables](https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables) if you dare to spend some time giving them some love. They suggest some good habits on how you should format your data which is pretty simple and makes a huge difference. These are not a list of must does to a table, but ideas worth keeping in mind for the next time a table is bothering you with how it looks. 
```

Reply:
How about this? (ammended to the second commit)